Author: Patrycja Rosa <alpine@ptrcnull.me>

	Add missing include for `_always_inline`
	This allows for building the Btrfs driver

diff --git a/filesystems/zstd/zstd_internal.h b/filesystems/zstd/zstd_internal.h
index f90ac42..013e6ff 100644
--- a/filesystems/zstd/zstd_internal.h
+++ b/filesystems/zstd/zstd_internal.h
@@ -26,6 +26,7 @@
 /*-*************************************
 *  Dependencies
 ***************************************/
+#include <linux/stddef.h>
 #include "error_private.h"
 #include "mem.h"
 #include "xxhash.h"
